administrator/language/de-DE/plg_multifactorauth_yubikey.ini

```
@@ -0,0 +1,17 @@
+; Joomla! German Translation
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MULTIFACTORAUTH_YUBIKEY="Multi-Faktor-Authentifizierung - YubiKey"
+PLG_MULTIFACTORAUTH_YUBIKEY_CAPTIVE_PROMPT="Bitte in das folgende Textfeld klicken. Dann den YubiKey in einen USB-Anschluss des Computers stecken und die goldfarbige Fläche berühren, um einen YubiKey-Code zu erzeugen. Wird ein NFC-fähiges Mobiltelefon oder Tablet mit einem NFC-fähigen YubiKey verwenden, muss der YubiKey stattdessen an den NFC-Lesebereich des Mobiltelefons oder Tablets gehalten werden."
+PLG_MULTIFACTORAUTH_YUBIKEY_CODE_LABEL="YubiKey-Code"
+PLG_MULTIFACTORAUTH_YUBIKEY_ERR_VALIDATIONFAILED="Es wurde kein gültiger YubiKey-Geheimcode eingegeben oder die YubiCloud-Server sind zu diesem Zeitpunkt nicht erreichbar."
+PLG_MULTIFACTORAUTH_YUBIKEY_LBL_AFTERSETUP_INSTRUCTIONS="Dieser YubiKey (welcher den Code erzeugt, der mit <code>%s</code> beginnt) wurde bereits eingerichtet. Auf dieser Seite kann nur der Titel geändert werden."
+PLG_MULTIFACTORAUTH_YUBIKEY_LBL_SETUP_INSTRUCTIONS="Bitte unten einen vom <a href='https://www.yubico.com/'>YubiKey</a> generierten Code eingeben und dann die Schaltfläche Bestätigen. Die ersten zwölf Zeichen, die den eindeutigen Identifikationscode für den YubiKey darstellen, werden gespeichert."
```

```suggestion
PLG_MULTIFACTORAUTH_YUBIKEY_LBL_SETUP_INSTRUCTIONS="Bitte unten einen vom <a href='https://www.yubico.com/'>YubiKey</a> generierten Code eingeben und dann die Schaltfläche Bestätigen. Die ersten zwölf Zeichen, die den eindeutigen Identifikationscode für den YubiKey darstellen, werden gespeichert."
```

rel noopen and noreferer

---

Ich hab einen Core PR erstellt.

https://github.com/joomla/joomla-cms/pull/38304